Gemfile

```diff
@@ -1,9 +1,9 @@
 source "https://rubygems.org"
 
-ruby "3.2.2"
+ruby "3.3.1"
```

Comment:
I recall there being issues with Render for Ruby 3.3 so would rather keep the Ruby version as it was and bump that separately.

Reply:
Upgraded to 3.2.4